clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -100,7 +104,7 @@ public JobWorkerImpl(
   }
 
   private void openStream() {
-    jobStreamer.openStreamer(this::handleJob);
+    jobStreamer.openStreamer(this::handleStreamedJob);
```

Comment:
Yes, see the linked issue: https://github.com/camunda/zeebe/issues/13814

It could be split off, but it was necessary for this feature to work properly, and I think small enough that it's fine to include here.